app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -986,9 +986,23 @@ $after-button-padding-left: govuk-spacing(4);
 
 @include govuk-media-query($media-type: print) {
   .gem-c-layout-super-navigation-header {
-    border-top: 1pt solid $govuk-print-text-colour;
-    margin: 0;
-    background-color: none;
+    border-top: 0;
+    border-bottom: 2pt solid $govuk-print-text-colour;
+    margin: 0 0 5mm;
+    background-color: transparent;
+
+    &:has(.gem-c-layout-super-navigation-header__header-logo .govuk-visually-hidden) {
```

Comment:
Appreciate I'm a bit late to this, but this feels like a brittle solution - I'd rather tie this to specific classes than assume a particular markup. I think it's also a little hard to read - it took me a bit of digging in the code to understand why this rule worked.

I think a better alternative would be to add a class somewhere to indicate when the header logo text is hidden, and also target the homepage class that already exists, something like:

```CSS
&.gem-c-layout-super-navigation-header--blue-background,
.gem-c-when-the-text-of-the-logo-is-hidden {
  border: 0;
  margin-left: 24px;
}
```

Reply:
I looked at this again because I didn't really see a problem with my original fix, but I've now noticed a different issue. We don't need the `visually-hidden` GOV.UK text at all and should remove it, then use your suggested fix instead.

**Longer explanation:**

There are two different logos that render depending on the value of `hide_logo_text` [link](https://github.com/alphagov/govuk_publishing_components/commit/6053c6d32fa2a78d7c8b33ed433c3f42a70f0e23).
-`hide_logo_text: false` Render a crown logo that inlcudes GOV.UK text
-`hide_logo_text: true` Render **only** a crown logo **without** GOV.UK text

Further logic renders a `visually-hidden` span containing GOV.UK text for the crown-only logo, but this seems redundant given that:

- Screen-readers read GOV.UK from the title of the SVG for pages with the crown logo + text.
- The Homepage displays "GOV.UK" text further down the page, clarifying the textless crown logo.

Also, there are some improvements to be made around how the GOV.UK text could be inserted in the SVG the same way as the `aria_label_text` is inserted in [_govuk_logo.html.erb](app/views/govuk_publishing_components/components/govuk_logo/_govuk_logo.html.erb)

This all needs further discussion, and given that my original fix has already been merged, it would be better to open a new issue for this within our team to try and address these issues collectively.

Reply:
Agreed. If you can condense all of the above into a draft card on the backlog, and we can look at it as a follow up piece of work in a later sprint, thanks.